core/invehicle-digital-twin/src/invehicle_digital_twin_config.rs

```diff
@@ -15,8 +16,13 @@ pub struct Settings {
 
 /// Load the settings.
 pub fn load_settings() -> Settings {
+    let config_filename_path = match env::var("IBEJI_HOME") {
+        Ok(s) => format!("{}/{}", s, CONFIG_FILENAME),
+        _ => CONFIG_FILENAME.to_owned()
+    };
```

Comment:
Please add this IBEJI_HOME matching statement to "/core/common/src/utils.rs" in the `load_settings<T>(config_filename: &str)` method.